gitnamehere/point-of-sale-group-project
# Create Payment Screen Pt. 1 (cont.)

**Ticket:** [Payment Screen](https://cs375-pos-group-project.atlassian.net/browse/ATS-20?atlOrigin=eyJpIjoiYzJlYzA2ZGI2MDY4NDExOWE0NWQwZWIyMmVkZmZjMzgiLCJwIjoiaiJ9)

- Confirgued `pay.js` to `script.js` for consistency and folder structure
- Added `index.html`, `styles.css`
- Added API endpoint to pull order details according to id
- Pulls subtotal according to order detail
- Added number pad for cash input
- Need to add process button
- Need to link payment page to order page
- Can use a bit more finer tweaking (tips, tax, and discount calculation bits, possible tab for credit cards etc)

Testing:
- `npm run setup-demo`
- `npm start`
- Go to `http://localhost:3000/pos/payment/?id={orderID}`
- Check if number "buttons" on number pad function normally (as if you were typing in a cash amount)
